blockchain/indexers/manager.go

```diff
@@ -301,16 +301,16 @@ func (m *Manager) Init(chain *blockchain.BlockChain, interrupt <-chan struct{})
 			return err
 		}
 
+		// Nothing to do if the index does not have any entries yet.
 		switch v := indexer.(type) {
 		case *SlpIndex:
 			if height == v.config.StartHeight {
```

Comment:
DIdn't we want to make this <= ?

Reply:
< would work, but this should never happen.

Reply:
Can we put in a comment on why that won't happen?

Reply:
ok, my mistake.  It does need the <, and the check needs to be move to the inner loop.  See 81d462c.